.happy/terraform/modules/batch/main.tf

```diff
@@ -123,6 +123,68 @@ resource aws_batch_job_definition dataset_metadata_update {
 })
 }
 
+resource aws_batch_job_definition rollback {
+  type = "container"
+  name = "dp-${var.deployment_stage}-${var.custom_stack_name}-rollback"
+  container_properties = jsonencode({
+  "command": ["python3", "-m", "backend.layers.processing.rollback"],
+  "jobRoleArn": "${var.batch_role_arn}",
+  "image": "${var.image}",
+  "memory": var.batch_container_memory_limit,
+  "environment": [
```

Comment:
hm good question--we don't need to, but it might be useful to clean-up. Actually, I imagine we already don't clean these up when we delete datasets in other scenarios. So this might be a general follow-up issue